qemu/tests/cfg/block_discard_hotplug.cfg

```
@@ -0,0 +1,37 @@
+- block_discard_hotplug:
+    no Windows
+    type = block_discard_hotplug
+    data_tag = "stg1"
+    images += " stg0 ${data_tag}"
+    force_create_image_stg0 = yes
+    image_size_stg0 = 1G
+    image_name_stg0 = images/stg0
+    drv_extra_params_stg0 += "discard=unmap"
+    boot_drive_stg1 = no
+    disk_serial = TARGET_DISK1
+    drv_extra_params_stg1 += "discard=unmap"
+    guest_mount_point = "/home/test"
+    guest_test_file = "${guest_mount_point}/test.img"
+    guest_format_command = "mkdir -p ${guest_mount_point};mkfs.ext4 {0} && mount {0} ${guest_mount_point}"
+    guest_dd_command = "dd if=/dev/zero of=${guest_test_file}"
+    guest_rm_command = "rm -rf ${guest_test_file};sync"
+    guest_fstrim_command = "fstrim ${guest_mount_point}"
+    variants:
+      - local_file:
+          force_create_image_stg1 = yes
+          image_size_stg1 = 1G
+          image_name_stg1 = images/stg1
+          blk_extra_params_stg1 += "serial=${disk_serial}"
+          # It need set discard=on the virtio-blk device under pc mode
+          virtio_blk..i440fx:
+            blk_extra_params_stg1 += ",discard=on"
+      - scsi_debug:
+          no virtio_blk
```

```suggestion
          only virtio_scsi
```
Maybe only virtio_scsi is more matching your case name `scsi_debug`.

---

Will update